Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Editor/ConfigurationChecker/Microsoft.MixedReality.WindowsMR.ConfigurationChecker.asmdef

```diff
@@ -1,15 +1,11 @@
 {
-    "name": "Microsoft.MixedReality.Toolkit.Providers.UnityAR",
+    "name": "Microsoft.MixedReality.WindowsMixedReality.ConfigCheck",
```

Comment:
@davidkline-ms, did you leave `.Toolkit.` out of these assemblies intentionally?

Reply:
I updated the asmdefs to include `.Toolkit.Providers.` like our other assemblies

Reply:
That was accidental. I was trying to limit the length of the asmdef file names and forgot to update the internal names.